src/main/java/AlexLink/HomeWork/HomeworkJDBS/DAO/SqlQueryOfSelect.java

```diff
@@ -0,0 +1,21 @@
+package AlexLink.HomeWork.HomeworkJDBS.DAO;
+
+public class SqlQueryOfSelect {
+
+    public static final String SELECT_TITLE_OF_MOVIES_BY_CURRENT_YEAR =
+            "SELECT title FROM movies " +
+            "WHERE EXTRACT(YEAR FROM premiere) = 2020 OR EXTRACT(YEAR FROM premiere) = 2021";
+
+    public static final String SELECT_INFO_ABOUT_ACTORS_IN_CURRENT_MOVIE =
+            "SELECT name_actors, birthday FROM actors " +
+            "WHERE name_actors IN (SELECT name_actor FROM listactors " +
+            "WHERE ID_movies = (SELECT id FROM movies " +
+            "WHERE title = ?))";
```

Comment:
Please read about SQL JOIN. This query can be:

```
SELECT a.name_actors, a.birthday FROM movies AS m
  INNER JOIN listactors AS la ON la.ID_movies = m.id
  INNER JOIN actors AS a ON a.name_actors = la.name_actor
WHERE m.title = ?  
```

Reply:
fixed